sympy/holonomic/holonomic.py

```diff
@@ -31,41 +31,50 @@
 
 
 def DifferentialOperators(base, generator):
-    """
+    r"""
+    This function is used to create annihilators using ``Dx``.
+
     Returns an Algebra of Differential Operators also called Weyl Algebra
-    and the operator for differentiation i.e. the `Dx` operator.
+    and the operator for differentiation i.e. the :math:`Dx` operator.
```

Comment:
This could probably be `Dx` between double backquotes, since this notation is typically used in code, not in mathematical texts. (Strings in single backquotes seem to be rendered by `latex`, though I have not found a good reference for that.)